internal/mode/static/nginx/config/upstreams.go

```diff
@@ -133,6 +172,12 @@ func (g GeneratorImpl) createUpstream(up dataplane.Upstream) http.Upstream {
 		Name:     up.Name,
 		ZoneSize: zoneSize,
 		Servers:  upstreamServers,
+		KeepAlive: http.UpstreamKeepAlive{
```

Comment:
I think you can just set this to `upstreamPolicySettings.KeepAlive`

Reply:
that doesn't work, http.Upstream still uses its own http.UpstreamKeepAlive type, maybe i missed something?

Reply:
hmm works for me

Reply:
😭  let me try again

Reply:
turns out @kate-osborn was right, added change